src/pid1.rs

```diff
@@ -21,12 +21,12 @@ pub struct Pid1Stats {
     pub tasks: u64,
 }
 
-/// Get procfs info on pid 1 - <https://manpages.debian.org/buster/manpages/procfs.5.en.html>
-pub fn get_pid1_stats() -> anyhow::Result<Pid1Stats> {
+/// Get procfs info on pid 1 - https://manpages.debian.org/buster/manpages/procfs.5.en.html
+pub fn get_pid_stats(pid: i32) -> anyhow::Result<Pid1Stats> {
```

Comment:
TIL linux PID table is still i32 limited ...

Reply:
The last time I checked PID_MAX_LIMIT was 2^22, so well within i32. I wonder if there's a use case for negatives here though.